docs/reference/installation.rst

```diff
@@ -49,7 +49,7 @@ Install without Symfony Flex packs
 
 Add ``SonataMediaBundle`` via composer::
 
-    composer require sonata-project/media-bundle
+    composer require sonata-project/media-bundle symfony/http-client nyholm/psr7
```

Comment:
IMO, we should explain why these extra dependencies are required, or be clear about if they are really mandatory or can be omitted under certain circumstances.

Reply:
Fixed